hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/replication/ECContainerReplicaCount.java

```diff
@@ -196,6 +196,29 @@ public Set<Integer> maintenanceIndexes() {
     return maintenanceIndexes.keySet();
   }
 
+  /**
+   * Gets a set containing all maintenance only indexes. These are replicas
+   * that are only on maintenance nodes, without any copies on healthy nodes.
+   *
+   * @param includePendingAdd true if any indexes that are pending an add
+   * {@link ContainerReplicaOp.PendingOpType#ADD} should be excluded
+   * @return set containing maintenance only indexes or empty set if none are
+   * in maintenance
+   */
+  public Set<Integer> maintenanceOnlyIndexes(boolean includePendingAdd) {
```

Comment:
I think we should remove the other version of maintenanceOnlyIndexes that returns a list, and make everything use this one. It keeps it consistent with decommissionOnlyIndexes